components/dash-core-components/package.json

```diff
@@ -60,6 +61,7 @@
     "react-resize-detector": "^6.7.6",
     "react-select-fast-filter-options": "^0.2.3",
     "react-virtualized-select": "^3.1.3",
+    "remark-math": "^3.0.1",
```

Comment:
We should decide which version to use.
I was able to use v1 and v2 as well; but not v4 and v5.
https://www.npmjs.com/package/remark-math?activeTab=versions

Reply:
Probably because we're on such an old version of `react-markdown`. Let's keep it at 3.0.1, it's going to be a substantial project (likely mostly on the QA side) to upgrade these.